docs/documentation/server_admin/topics/authentication/flows.adoc

```diff
@@ -403,6 +403,25 @@ Note when the login request initiates a request with the `claims` parameter requ
 one of the specified levels. If it is not able to return one of the specified levels (For example if the requested level is unknown or bigger than configured conditions
 in the authentication flow), then {project_name} will throw an error.
 
+[[_registration-rc-client-flows]]
+==== Registration or Reset credentials requested by client
+
+Usually when user is redirected to the {project_name} from client application, the `browser` flow is triggered. This flow may allow user to <<con-user-registration_{context}, register>> in case
+that realm registration is enabled and user clicks to the link `Register` on the login screen. Also in case that <<enabling-forgot-password, Forget password>> is enabled for the realm, user can
+click on the link `Forget password` on the login screen, which will trigger `Reset credentials` flow where user can reset his credentials after confirmation of his email address.
+
+Sometimes it can be useful for client application to directly redirect user to the *Registration* screen or to *Reset credentials* flow. This will have exactly same behaviour like when manually
+user clicks *Register* or *Forget password* links on the normal login screen. Automatically redirect to registration or reset-credentials screen can be done as follows:
+
+* When client wants user to be redirected directly to the registration, OIDC client should replace very last snippet from the OIDC login URL path (`/auth`) with `/registrations` . So the full URL
+might be something like `https://your-keycloak-host/realms/your_realm/protocol/openid-connect/registrations`.
+
+* When client wants user to be redirected directly to the `Reset credentials` flow, OIDC client should replace very last snippet from the OIDC login URL path (`/auth`) with `/forgot-credentials` .
+
+WARNING: The steps above are only officially supported way how client can directly request registration or reset-credentials flow. For security
```

Comment:
```suggestion
WARNING: The steps above are only officially supported way how client can directly request registration or reset-credentials flow. For security
```
```suggestion
WARNING: The preceding steps are the only supported method to how a client can directly the request registration or reset-credentials flow. For security
```

Reply:
I removed "officially" because it sounds as if there "unofficially" supported methods.